docs/administering/offline.md

```diff
@@ -0,0 +1,100 @@
+
+You can obtain the latest version of the installer script from our GitHub repository: [install.sh](https://raw.githubusercontent.com/sandstorm-io/sandstorm/master/install.sh) Transfer this file to your server as well.
+
+Now, on your server, you can run:
+
+    bash install.sh sandstorm-<version>.tar.gz
+
+(Replace `<version>` with the version number you downloaded.)
+
+Follow the on-screen instructions to set up your server. Note that:
+
+- You cannot use the Sandcats DNS service. You must configure your own DNS. Remember that [Sandstorm requires a wildcard DNS entry](wildcard.md).
+- You must use LDAP or SAML login to integrate with your network's single sign-on. Google, Github, and (probably) e-mail login will not work since they require internet access.
+- When you reach the step in the installer where Sandstorm wants to pre-install some apps, choose "Skip for now", since Sandstorm won't be able to reach the app server.
+
+## Updating Offline
+
+Normally, Sandstorm automatically updates itself. Of course, it cannot do this if it does not have access to the internet. You should disable automatic updates by editing `/opt/sandstorm/sandstorm.conf`. Find the line for `UPDATE_CHANNEL` and change it to:
+
+    UPDATE_CHANNEL=none
+
+Then restart Sandstorm:
+
+    sudo sandstorm stop
+    sudo sandstorm start
+
+Now, to update Sandstorm manually, obtain the latest release tarball from `dl.sandstorm.io` the same as you did to install it originally. Upload this to your server.
+
+Then, run:
+
+    sudo sandstorm update sandsotrm-<version>.tar.gz
+
+## Installing Apps
+
+To install an app, you must manually obtain the app's SPK package file, then upload it to your server through its web interface. To obtain an SPK package from the app market, [visit the market](https://apps.sandstorm.io), find the app you want, and click the "Download SPK" button in the upper-right of the app's page. This will download the SPK file.
+
+Unfortunately, the file will download with a name that is a long string of letters and numbers and does not end with `.spk`. You will need to rename the file to end with `.spk` before you can upload it to Sandstorm.
+
+Once you have the SPK file and have renamed it, browse to your Sandstorm server, go to "Apps" in the left sidebar, and click the "Upload app..." button in the upper-right corner. Choose your SPK file.
+
+Note Sandstorm is designed such that every user has an independent workspace and can upload their own apps. Every user will need to upload their own copy of the SPK for each app they wish to use. If you'd rather manage applications and updates centrally, you will need to run an offline app index (described below).
+
+## Updating Apps
+
+Normally, Sandstorm automatically checks the app market for updates, notifies you when an update is available, and lets you install the update with one click. Without internet access, it won't be able to do so.
```

Comment:
for precision: "checks the app index" (but you haven't introduced that idea yet) or "checks `sandstorm.io`"